src/cartesian/getEquidistantTicks.ts

```diff
@@ -35,7 +35,7 @@ export function getEquidistantTicks(
 
     const tickCoord = entry.coordinate;
     // We will always show the first tick.
-    const isShow = index === 0 || isVisible(sign, tickCoord, size, start, end);
+    const isShow = index === 0 || isVisible(sign, tickCoord, () => size, start, end);
```

Comment:
Is the tick size not already computed in line 34? 
If I understand correctly, this should be 

```
    // Check if the element collides with the next element
    const getSize = () => getTickSize(entry, index);

    const tickCoord = entry.coordinate;
    // We will always show the first tick.
    const isShow = index === 0 || isVisible(sign, tickCoord, getSize, start, end);
```

Now we would not call getTickSize directly.

Reply:
Doing so triggers the following lint error:

![image](https://github.com/recharts/recharts/assets/12778398/131f0905-0a0f-48f1-a300-ac298e9f2315)

But I think I managed to find a workaround. Pushing a new commit 😄